README.md

```diff
@@ -1,75 +1,88 @@
+
+Check out [mozilla/shield-studies-addon-template/](https://github.com/mozilla/shield-studies-addon-template/) to get started with an example study where shield-studies-addon-utils is already installed and configured.
+
+## Installing the `StudyUtils.jsm` in your add-on
 
 ```
 npm install --save-dev shield-studies-addon-utils
 ```
 
-Copy the file to somewhere useful in your addon.
-
-
-## Tutorial and Full Usage
-
-[fully worked tutorial - How To Shield Study](./howToShieldStudy.md)
-
-## Example embedded web extension study
-
-See [examples](https://github.com/mozilla/shield-studies-addon-utils/tree/master/examples) directory.
+Copy `dist/StudyUtils.jsm` to your `addon` source directory, where it will be zipped up.
 
 ## Summary
 
 ### Design Case
 
 Your Study is:
 
-- side-by-side variations (1 or more)
+* side-by-side variations (1 or more)
 
 ### Benefits
 
 Using this, you get this analysis FOR FREE (and it's fast!)
 
-- Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
-
-
-## Development
+* Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
 
-- open an issue
-- hack and file a PR
+## Engineering and Process
 
+* [Shield article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield)
+* [Shield Studies article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies)
+* [mozilla/shield-studies-addon-template/](https://github.com/mozilla/shield-studies-addon-template/)
+* [Current work-in-progress docs and launch process](https://github.com/mozilla/shield-studies-addon-utils/issues/93)
+* [Long, rambling engineering docs](./docs/engineering.md)
+* Come to slack: #shield
 
 ## Gotchas, Opinions, Side Effects, and Misfeatures
 
-1.  No handling of 'timers'.  No saved state at all (including the variation name), unless you handle it yourself.
+1.  No handling of 'timers'. No saved state at all (including the variation name), unless you handle it yourself.
 
 2.  No 'running' pings in v4 (yet).
 
-3.  User disable also uninstalls (and cleans up)
+3.  User disable also uninstalls (and cleans up).
+
+## Development on the Utils
+
+* open an issue
+* hack and file a PR
+
+## History of major versions
 
+* v4.x: (proposed) additional functions for common cases
+* v4: First `.jsm` release. Uses packet format for PACKET version 3.
+* v3: Attempt to formalize on `shield-study` PACKET version 3. Jetpack based. Prototype used for `raymak/page-reload`. All work abandoned, and no formal npm release in this series. Work done at `v3-shield-packet-format` branch. LAST JETPACK (addon-sdk) RELEASE.
+* v2: Code refactor to es6 `class` with event models. Added cli tooling. Packet format is still arbitrary and per-study. Jetpack based. Last used in studies in Q2 2017.
+* v1: Initial work and thinking. Telemetry packets are rather arbitrary. Jetpack based.
 
-[example]: examples/README.md
+### Legacy template repositories
```

Comment:
Hopefully once we have updated the template and utils repos, we can remove this section. But good to warn people for now...

Reply:
We can remove this section when confusing repositories stop showing up in google results for new engineers :)